### PR TITLE
Created server and select and insert requests for postgres

### PR DESCRIPTION
**Pull request by kcmcminn**

### CODE REVIEW

File: Server/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    results.rows.forEach((result) => {
+      result.createdAt = result.createdat.toISOString().substring(0, 10);
+      result.author = result.username;
+    })
+    res.send(results.rows)
+  });
+})
+
+app.post('/api/reviews/:productid', (req, res) => {
+  console.log('here', req.body)
```
Comment (mmeigooni) [inline Server/index.js]: Is this necessary?

File: database/controllers/reviews.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const client = require('../index.js');
+
+module.exports = {
+  getAll: (productId, callback) => {
+    var selectQuery = `
+    SELECT * FROM reviews WHERE productId = ${productId}
```
Comment (mmeigooni) [inline database/controllers/reviews.js]: Indention

**Review by mmeigooni — APPROVED** [review]
Easy review! Everything looks great, made a couple tiny suggestions.

File: Server/index.js (hunk 1 above)
Comment (kcmcminn) [reply Server/index.js]: Thanks for the catch. I'll delete it.

File: database/controllers/reviews.js (hunk 2 above)
Comment (kcmcminn) [reply database/controllers/reviews.js]: Thank you! I'll fix this too.

**Pull request merged**